docs/reference/reference-deploy/config/master-config-reference.rst

```diff
@@ -113,6 +113,9 @@ The master supports the following configuration settings:
       ``/var/cache/determined``. Note that the master would break on startup if it does not have
       access to create this default directory.
 
+-  ``launch_error`` (optional): Specify whether to refuse an experiment or task if the slots
+   requested exceeds the cluster capacity. Defaults to ``false`` and shows a warning instead.
```

Comment:
let's default to true, i.e. error.  
- if it turns out to be broken for some users, they can opt out
- if it works 100% of the time and everyone loves it, we remove this option